pyvista/plotting/themes.py

```diff
@@ -3058,7 +3059,7 @@ def save(self, filename: str) -> None:
         data = self.to_dict()
         # functions are not serializable
         del data["before_close_callback"]
-        with open(filename, 'w') as f:
+        with Path(filename).open('w') as f:
```

Comment:
I'm really not a fan of these changes... `open(filename, 'w')` has be the pattern for ages. I don't love seeing this well established convention replaced.